content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -581,6 +554,58 @@ Volume binding matrix for statically provisioned volumes:
 **Note:** Only statically provisioned volumes are supported for alpha release. Administrators should take care to consider these values when working with raw block devices.
 {{< /note >}}
 
+## Types of Persistent Volumes
+
+`PersistentVolume` types are implemented as plugins.  Kubernetes currently supports the following plugins:
+
+* GCEPersistentDisk
+* AWSElasticBlockStore
+* AzureFile
+* AzureDisk
+* FC (Fibre Channel)**
+* FlexVolume
+* Flocker
+* NFS
+* iSCSI
+* RBD (Ceph Block Device)
+* CephFS
+* Cinder (OpenStack block storage)
+* Glusterfs
+* VsphereVolume
+* Quobyte Volumes
+* HostPath (Single node testing only -- local storage is not supported in any way and WILL NOT WORK in a multi-node cluster)
+* Portworx Volumes
+* ScaleIO Volumes
+* StorageOS
+
+Raw Block Support exists for these plugins only.
+
+### Manually provisioning a GCE Regional Persistent Disk
```

Comment:
Yeah I'm not sure about putting it here.  I think it is fine to put it in the Volumes page (I put local PV stuff there)

Reply:
Problem was that the Travis test wouldn't allow it in the Volumes page.

Reply:
How is it allowing local PV to be in the volumes page?

Reply:
Was it enabled recently in Travis?

Reply:
Not sure why the local PV example is not triggering a Travis error. The current version of [examples_test.go](https://github.com/kubernetes/website/blob/master/test/examples_test.go#L628) does not include the `PersistentVolume` kind for `/content/en/docs/concepts/storage/volumes.md`, just `Pod`.

Reply:
Turns out it's because there's a space in the code block declaration: ` ``` yaml`